src/io/git.go

```diff
@@ -0,0 +1,78 @@
+package io
+
+import (
+	"os"
+	"os/exec"
+	"sort"
+	"strings"
+)
+
+//GetTemplateVersion checkouts either the latest compatible version or the user specified version
+//Used to make sure the current versionof the cli can always use a newer compatible template version, without begin updated
```

Comment:
checkouts -> check outs
versionof -> version of
without begin updated -> without being updated (?)